### ISSUE TITLE
Highlight active item in main menu

### ISSUE THREAD

**Issue by ms-studio** [coursweb/jk-theme#8]
This was previously done in Liquid.

```
{% assign page_url_parts = page.url| split: '/' %}

{% if chapter.url == page_url_parts[1] %}  active{% endif %}
```

But as we don't provide anymore the URL path for every page, this cannot work anymore. Liquid "does not know anything about URLs or address bars or websites and such". 

The highlighting can be done via Javascript. Or maybe there are some variables available? Actually yes, here: 
https://github.com/jekyll/github-metadata

One interesting variable that could do the trick: `site.baseurl` (the project name for project pages if none is set)

**Comment by ms-studio** [comment]
Observations:

We combine two variables: ``{% assign page_url_this = site.url | append: site.baseurl %}``

The result:

- `site.url` produces "http://cours-web.ch" – no https because of bad DNS configuration at host level...
- `site.baseurl` produces "/media" – note that there is no trailing slash.

**Comment by ms-studio** [comment]
Fixed by 66d0a673d8740a8b9a66d7dba808b9b8406e2705
